src/hii_popdens.py

```diff
@@ -11,8 +11,19 @@ class HIIPopulationDensity(EETask):
             "ee_type": EETask.IMAGECOLLECTION,
             "ee_path": "CIESIN/GPWv411/GPW_Population_Density",
             "maxage": 5  # years
-        }
-    }
+        },
+        "ocean": {
+            "ee_type": EETask.IMAGE,
+            "ee_path": "users/aduncan/cci/ESACCI-LC-L4-WB-Ocean-Map-150m-P13Y-2000-v40",
```

Comment:
All sources that are not available in ee catalog or are the results of other tasks in the pipeline should be moved into `projects/HII/v1/source`